.github/workflows/pr_tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     strategy:
```

Comment:
Still points at main!

Reply:
I realised, created the new branch, then forgot to change it anyway, thank you!